internal/graphapi/controlscheduledjob.resolvers.go

```diff
@@ -0,0 +1,164 @@
+package graphapi
+
+// This file will be automatically regenerated based on the schema, any resolver implementations
+// will be copied through when generating and any unknown code will be moved to the end.
+// Code generated by github.com/99designs/gqlgen
+
+import (
+	"context"
+
+	"github.com/99designs/gqlgen/graphql"
+	"github.com/rs/zerolog/log"
+	"github.com/theopenlane/core/internal/ent/generated"
+	"github.com/theopenlane/core/internal/ent/generated/controlscheduledjob"
+	"github.com/theopenlane/core/internal/ent/generated/scheduledjob"
+	"github.com/theopenlane/core/internal/graphapi/model"
+	"github.com/theopenlane/utils/rout"
+)
+
+// CreateControlScheduledJob is the resolver for the createControlScheduledJob field.
+func (r *mutationResolver) CreateControlScheduledJob(ctx context.Context, input generated.CreateControlScheduledJobInput) (*model.ControlScheduledJobCreatePayload, error) {
+	// set the organization in the auth context if its not done for us
+	if err := setOrganizationInAuthContext(ctx, input.OwnerID); err != nil {
+		log.Error().Err(err).Msg("failed to set organization in auth context")
+
+		return nil, rout.NewMissingRequiredFieldError("owner_id")
+	}
+
+	// by default copy the cadence or cron from the "scheduled job" template
+	// if this is not provided here
+	query, err := withTransactionalMutation(ctx).ScheduledJob.Query().
+		Where(scheduledjob.ID(input.JobID)).
+		CollectFields(ctx)
+	if err != nil {
+		return nil, parseRequestError(err, action{action: ActionGet, object: "scheduledjob"})
+	}
+
+	scheduledJob, err := query.Only(ctx)
+	if err != nil {
+		return nil, parseRequestError(err, action{action: ActionGet, object: "scheduledjob"})
+	}
```

Comment:
I dont think you need `CollectFields` here - this is used to limit the columns returned from a graph request; but since you are writing the query you can simply do:

```suggestion
	// by default copy the cadence or cron from the "scheduled job" template
	// if this is not provided here
	scheduledJob, err := withTransactionalMutation(ctx).ScheduledJob.Query().
	        Select("candence", "cron"). // just select the fields you want
		Where(scheduledjob.ID(input.JobID)).
		Only(ctx)
	if err != nil {
		return nil, parseRequestError(err, action{action: ActionGet, object: "scheduled_job"})
	}
```